.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,70 @@
+<!--
+Thanks for filing an OSCC issue! You're making our ecosystem a better place!
+
+Below are some templates to get you started, filling out any that are relevant
+to this issue will help us a lot! Feel free to delete any information or text
+that isn't relevant to your issue liberally.
+-->
+
+<!-- Questions about purchasing and pricing for OSCC boards or DriveKit?
+Shoot an email to drivekit@polysync.io -->
+
+
+
+<!--
+Technical issue template.
+
+Are you having a technical issue? Fill in the following blocks to help us
+tackle it!
+-->
+
+### Expected behavior
+<!-- What should be happening? -->
+
+### Actual behavior
+<!-- What behavior are you seeing? -->
+
+### Steps to reproduce
+<!-- How can we make it happen too? -->
+
+### Version info
+_What's the output of running `git describe --tags` in your OSCC directory?_
```

Comment:
Should we ask explicitly for both firmware version and API version since it is possible they're not the same?

Reply:
@rebpdx any chance you can recommend some phrasing to indicate that?

Reply:
Since we currently don't output firmware version through any messages or anything it requires knowing what you flashed onto the hardware. Maybe for now we just add a second question along the lines of:

> Is this the same version flashed onto the hardware?